solutions/18.py

```diff
@@ -0,0 +1,20 @@
+#Himanshu Nachane (@Himanshu1495)
+'''return missing number in an input array of consecutively increasing numbers.
+
+i: [5,6,8,9]
+o: 7
+'''
+
+def missing_num(arr):
+	#get first element of array
+	start = arr[0]
+	#get last element of array
+	end = arr[-1]
+	#create new array by iterating from start element to end element
+	new_arr = [x for x in range(start,end+1)]

```

Comment:
What you are trying to do here is add up all the numbers from a to b. You don't need an array to do this (for loop? While loop? math formula?)